docs/exposing-eventlisteners.md

```diff
@@ -32,9 +32,10 @@ services can be found
       # Check https://kubernetes.github.io/ingress-nginx/deploy for other installation options
       kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.34.1/deploy/static/provider/cloud/deploy.yaml
    ```
-2. Find the service name expose by the Eventlistener service:
+2. Find the service name expose by the Eventlistener service (assuming
+   it is under namespace `getting-started`):
    ```sh
-    kubectl get el <EVENTLISTENR_NAME> -o=jsonpath='{.status.configuration.generatedName}'
+    kubectl get el <EVENTLISTENR_NAME> -n getting-started -o=jsonpath='{.status.configuration.generatedName}{"\n"}'
```

Comment:
Thanks for adding this. Usually in most of our examples/snippets, we generally omit the namespace parameter since it is a common parameter that applies to almost all kubectl commands. However, I do realize that it is confusing for users following the `getting-started` tutorial since it uses a non-default namespace. What do you think of having both i.e. one snippet with no namespace with a note/comment below with an example mentioning that users should add a `-n <namespace>` param for their commands if needed?

Reply:
I agree with you. I realized people might be coming here from other places beside the tutorial. But then I saw the yaml right after this that installs Ingress in getting-started namespace. So I thought maybe not. 
But I'm totally fine with your solution. what should we do with the Ingress resource?

Reply:
I'm fine with a quick note there to indicate that namespace might be needed too!